sdcm/tester.py

```diff
@@ -291,12 +291,15 @@ def __init__(self, *args):  # pylint: disable=too-many-statements
             Setup.configure_rsyslog(self.localhost, enable_ngrok=False)
 
         self.alternator: alternator.api.Alternator = alternator.api.Alternator(sct_params=self.params)
-        if self.params.get("use_ldap_authorization") or self.params.get("prepare_saslauthd") or self.params.get(
-                "use_saslauthd_authenticator"):
+        if (self.params.get("use_ldap_authorization") or self.params.get("prepare_saslauthd") or self.params.get(
+                "use_saslauthd_authenticator")) and not self.params.get("use_ms_ad_ldap"):
             self.configure_ldap(node=self.localhost, use_ssl=False)
+        if self.params.get("use_ms_ad_ldap"):
```

Comment:
this could be inverted, and be like:
```
if self.params.get("use_ms_ad_ldap"):
...
elif self.params.get("use_ldap_authorization") or self.params.get("prepare_saslauthd") or self.params.get("use_saslauthd_authenticator"):
...
```

Reply:
I agree, it's more readable like Fabio suggested and less error prone.